### DISCUSSION TITLE
Suggestion for small logo adjustments for better balance

### DISCUSSION THREAD

**Discussion by Flavelius** [godotengine/godot-proposals#7304] [General Discussions]
While the current logo is certainly usable it still has that small notion of unwilling derpiness that's one of the small parts with potential for improvement for the aspiration of engine adoption by larger studios which are aware that the choice of engine also includes its marketability.

With 4.0 onwards godot now seems to have taken a leap towards being more professional, maturing in a way.  
And this, i believe should be reflected in the logo aswell, which is one of the more prominent parts of public representation.

To address that hint of unwilling derpiness, not coming up with yet another logo redesign, i just attempted to balance out certain parts of the current design, and this is the result:

![Adjusted](https://github.com/godotengine/godot-proposals/assets/8841352/e45bc92d-89e8-4f9e-990a-335c4d2d8ec3)

While it doesn't look like much changed, which is intended, it's in the details that are better visible this way:

![animation](https://github.com/godotengine/godot-proposals/assets/8841352/c4103bab-6c9b-4086-b9ee-40b341b92c13)

This balances out the upper and lower part a bit more and un-squashes it slightly, aligning some of the invisibly extended geometry just a little better into the imagined circle it all sits in.

[Adjusted.zip](https://github.com/godotengine/godot-proposals/files/12064423/Adjusted.zip)

**Comment by AThousandShips** [comment]
I think a more clear reasoning rather than personal preference and claims of professional look is warranted here

**Reply by Flavelius** [reply]
I'm beginning to think that you're on a personal crusade against me. Why is it that you're always the first to comment and always so negative towards my proposals while dismissing my words as being mere valueless claims (you mainly assume i have no professional background)?

**Reply by AThousandShips** [reply]
What? I'm not targeting you in any way, why do you interpret me commenting on your post first as some kind of personal attack?

I am not dismissing anything here, you have made no clear arguments here about what makes this design more professional, I have no idea of your professional background, and it hardly matters. Just saying "it's more professional" without backing that with some tangible "these are professional design clues" makes it appear as nothing but personal preference. That's literally why I am asking for some points on how it is tangible

Stop taking feedback as some personal attack and imagining that I have some personal vandetta against you, I react and interact with dozens of things every day

**Reply by AThousandShips** [reply]
Accusations of personally targeting you are unprofessional and inappropriate, please stop

They are baseless and I resent the accusations of me being unprofessional

**Reply by Flavelius** [reply]
Well, i wrote what this does, it's balancing the weight of the upper part with the lower part just a bit more, quantitatively. It also unsquashes the base shape so it fit's a bit more into the imagined circle the whole logo sits in.
I'm open to suggestions for finding the right words that you deem professional.
And i'm not accusing you of anything directly, neither did i call you unprofessional anywhere, i just said what i was beginning to think and what lead me there with the question directed to the negativity contained in your posts to determine if that line of thought was warranted. This seems to be taken as an accusation, so i'm being called unprofessional again, i hope you understand that this further devalues not just my entry post but my comments aswell and is detrimental to my motivation to engage in supportive feedback on this platform.

**Reply by AThousandShips** [reply]
> This seems to be taken as an accusation, so i'm being called unprofessional again, i hope you understand that this further devalues not just my entry post but my comments aswell and is detrimental to my motivation to engage in supportive feedback on this platform.

It is unprofessional to jump to this conclusion without any reason, it doesn't matter that you made the comment in vague terms, you presented the image that I was responding to you based not in neutral criticism but in personal feelings. This pains a bad image of me, so you claiming that *I* am detrimental to *your* image when you do this is frankly absurd.

I have not attacked your professionalism, I have questioned it since it *is* unprofessional to behave as you do. Again, saying "I'm beginning to think that X" is just plausible deniability about direct accusations.

I'll expect an apology for your unprofessional attitude, your insulting behaviour, and your attempt to paint me in negative and unprofessional terms. I do not accept attempts to slander my professionalism.

Other than that I won't engage any further with you. I would have given my feedback, but you wouldn't let me.

**Reply by Flavelius** [reply]
>It is unprofessional to jump to this conclusion without any reason, it doesn't matter that you made the comment in vague terms, you presented the image that I was responding to you based not in neutral criticism but in personal feelings

The same can be said about your entry comment, it's presenting the image that the entry post is personal preference presented as claims of professionality, which as you know implies being deceptive. In my unprofessional personality, i don't like this being implied about me, yes. As i remembered our last discussion where it was a back and forth of negatively connotated misunderstanding which i had to stop replying to as i found it was not being productive anymore, i wanted to be clear that you're not just continuing that sentiment here, which was the basis for my assumption and the following question.
I can only appologize so far as to be sorry that this again took a similar turn as our last discussion because of mutual misunderstanding.
If you think i bear the sole responsibility for it, so be it, but unrelated to my actual professional background was the initial post not about deceptively claiming that this was more professional, just about adressing one kind of impression the logo gives that i deemed improvable to match the way that i experience godot to be wanting to present itself (more professionally, looking to address larger studios).
Either way, i understand your conclusion and think it's better not to continue this.

**Reply by betalars** [reply]
from a professional perspective I have to say: I really like the cog-gears a lot more in the new proposal, I think they have a much clearer silhouette while staying true to the original.

I think all other changes are detremental tho, as they make it more generic in my opinion. I like the Godot Logo having a character quite a lot and I would only change it for something that keeps that.

**Comment by LikeLakers2** [comment]
I'm against the proposed changes to the logo. In my opinion, the "unwilling derpiness" gives the logo a bit of personality. It makes it look like a mascot for the engine - whereas removing that "unwilling derpiness" makes it look like it was made solely as a logo. And personally, I would rather have it look a little silly than be strictly professional.

**Reply by Flavelius** [reply]
That does make sense. I based my assumption on the changes with 4.0 and the beginning of the godot foundation and the official sentiment in communication, in that the whole endeavour has become a little more curageous about adressing larger professional studios, which tend to assign a little more value to the presentability of their tech stack. But i see the value in the derpiness-charm for its current community that i guess learned to like it for not copying the hollow marketability of other engines.

**Comment by Calinou** [comment]
While I think your changes look good, one issue with subtle changes to a logo is making sure that all third-party usages are up-to-date.

If you look around, you may still find a few resources using the Godot 2.x logo, despite the logo having changed to its current design in January 2018. This will be even more of an issue with a change that is unnoticeable to the viewer in anything else than a side-by-side comparison.

As a result, it may be better to stick with a technically "worse" logo but making sure that third-party usages are more consistent in the process. We do have a [press kit](https://godotengine.org/press/) on the website, but not everyone will be checking it regularly after putting a Godot logo on a website, video thumbnail or project splash screen.

**Reply by Flavelius** [reply]
Yes, that's very reasonable, i can see that this is too small of a change with not enough requirements to warrant all the following discrepancies.

**Comment by CarpenterBlue** [comment]
What if we give it cat ears to make it UwU? I guarantee that would make people to use the new logo.
![image](https://github.com/godotengine/godot-proposals/assets/114881518/1d00533e-fb53-4914-a578-2bc3161290ce)
That said, this is a nice redesign.

**Comment by xdega** [comment]
I want to just add my 2c, the logo is bad. You can call it "opinion", but I am clearly not the only one if there have been many discussions (promptly shut down) that share the same concern. Opinion still matters. Even if it is the "incorrect" way to judge a tool, people will still be turned away if something looks outright unappealing or unprofessional.

Now, Godot may be on the verge of becoming much more popular than before. I think a more modern logo design would be ideal if the leadership would simply entertain the discussion, instead of shutting down any suggestion in favor of the current iteration that has a very dated, amateur, look to it.

I am sure there are lots of very talented designers who would jump at the opportunity to re-imagine the branding for this project.

**Reply by xdega** [reply]
I will say that the foundation logo is much better, and the "modern" look that may be preferable for the engine itself:
![image](https://github.com/godotengine/godot-proposals/assets/8093386/6fa99c40-ccbc-45f2-adf2-00d1ebb344b2)

**Reply by Flavelius** [reply]
While i also think this logo looks good and professional I'm also a little surprised that this was chosen for the foundation as it shares its characteristics with those of another game engine's/organization's logo (leadwerks}

**Reply by CarpenterBlue** [reply]
Open source projects have absolute disdain for graphic designers.
To the point that it became meme when communities start talking about starting an open source project, the idea is shut down with someone saying jokingly "I will make the logo" because it happens so often and because designers are so eager to contribute it.
This mentality really extends to all of the UI design and it's really the reason why most of open source projects look so dated... they are made by programmers who have more important things to do and do see any value in things being pretty.
It's tradition to reject any offer to change the logo as it happens 30 times a minute for any major project no matter the quality of the actual logo.
Yes, Godot's logo is amateurish. I would have picked one by Kenney myself but as it stands, this is open source project and designers are by tradition not welcome to contribute. All you can do is meme about it because of the stigma that a pretty logo is a bloat and doesn't bring any actual value to the software and other various mindsets.

I am not happy about this mentality, it drives lot of creative people away. I seen some extremely professional UI redesigns that were rejected and rightfully so, it's true that the functionality comes first but the pretty coat is really a powerful thing. People do judge a book by it's cover.

It's especially valuable in these times when there is lot of eyes looking to switch from their main engine.

**Reply by xdega** [reply]
I can't speak for which engine is better, but another cross platform game engine that could be an option for many has, in my opinion, more appealing branding. If both engines are going to cost the same overhead and the pros/cons are a wash... it leaves the judgement to be upon things like "are they open to creativity?", "does it look professional?", etc...

So we can look at things like this...
![logo-ver-classic-white-160](https://github.com/godotengine/godot-proposals/assets/8093386/468470b4-80f5-41fe-9023-932d26f9348b)

versus this...
![6318500](https://github.com/godotengine/godot-proposals/assets/8093386/dc2338cd-f86b-42bf-bd83-f2aeb24ca519)

Their associated websites, and associated communities and tooling. From what little I have seen so far, the maintainers have been quick to shut down discussion on the logo. Why not leave the discussion open? (I guess it is sorta here in the forums, just not in the repo issues) Perhaps there are devs who are at the intersection of graphic arts and engineering, who could provide valuable insight. 

I personally lean towards the frontend, but I am not a logo designer. I do, however, know enough to know that a well designed logo is not just a "pretty picture", it is a designed artifact to convey a message. The message that the Godot logo conveys has been stated by multiple people that the engine is "amateurish", "childish", "unprofessional". These are not good adjectives, IMO.

**Reply by Flavelius** [reply]
I share these concerns, but i also understand that they shouldn't change the logo at some random point as it's integral to the public representation through association. I think they just missed that point with 4.0 which was publicly touted as the point of intentional incompatibility with focus on big improvements which could have been the point to also reevaluate the logo beside other things, but it was obvious that there was/is not enough manpower in the core team that controls most if not all aspects of change in the engine to keep focus and work on all those aspects of the engine at the same time as even some of the core features were postponed to 4.1 etc. I found myself thinking about proposing Godot to be used at my company in the past aswell; it's hard to sell at this point though and that is not just based on professional looks, shiny things sell better, yes, but prioritizing core and features is a good decision as those can be argued for when looking past the representational side. Maybe they will be able to reevaluate even a slight adjustment to all the CI/CD at some point in the future.

**Reply by timothyqiu** [reply]
I think the adjusted version looks better.

But I personally love the current one _because_ it's cute and non-professional, just like the Linux Tux. I see "amateurish" and "childish" as a compliment.

**Reply by xdega** [reply]
I think people should stop comparing Godot to the Linux logo. That logo was created in 1996, amid the .com boom, at a time where tech was much more of a "hobby" than it is today. It has grown into an iconic image, born at a time when aesthetics were much different. 

I could guarantee you that Linux wouldn't have a logo like that if the project only just started (besides, lots of distros have their own, modern, logos).

I am all for a mascot, but a mascot is not a logo. Rust the programing language has a cute one: 
![image](https://github.com/godotengine/godot-proposals/assets/8093386/88b084fb-0175-4b21-b6a0-4eab5afd12bd)

The issue that some people have brought up, is that the Godot logo isn't a logo you want to proudly display in marketing materials, websites, etc. This in turn will limit the exposure of the engine (once this initial hype dies down), and in turn could result in the loss of would-be contributors. I'll be quite honest, I a not sure I would even want to display the Godot logo in a personal portfolio. It looks like a high-school project created in MS Paint to put it bluntly.

... all of that aside, I did notice that there would be a heavy lift to update all the documentation, style guides, etc. It honestly confuses me that such a logo even has a "style guide", that time would have been better spent brainstorming something fresh, IMO.

**Reply by timothyqiu** [reply]
> I could guarantee you that Linux wouldn't have a logo like that if the project only just started

> I a not sure I would even want to display the Godot logo in a personal portfolio. It looks like a high-school project created in MS Paint to put it bluntly.

These are based on the idea that using a professional/modern looking logo is better. Yet you cite these arguments in order to support this point. I think that's a circular argument and not helpful to this discussion.

> the Godot logo isn't a logo you want to proudly display in marketing materials, websites, etc

The same applies to this one. However, this argument is not exactly true either. Developers are given the option to disable/customize boot splash in project settings. But many, at least the people I know, choose to keep it proudly or derivative base on the game's theme.

> besides, lots of distros have their own, modern, logos

This is supported as said above.

**Reply by xdega** [reply]
First off, I want to say thank you for engaging in this discussion. Second of all, I can't say I really follow your lines of rebuttal. Maybe you are misunderstanding my arguments? I acknowledge that my post was a bit wordy and, as a result, the core points I was making may have been lost. So here they are in bullet point form.

- Comparing to Linux logo is comparing to an old logo from the 90s... this supports the sentiment that the current logo is dated. 

- Having a cute mascot is good. But the Logo doesn't need to be that mascot. (that being said, @Dark-Horizons responded with a great example of how you might include the mascot in a logo)

- I assert that a professional/modern logo in 2023 is better. Why? Because what I mean by a "professional" logo is one that is designed to universally communicate a positive message. The current logo doesn't seem to do that at all. People will often just a book by it's cover (logo), and that's unfortunate, but a harsh reality in a saturated "market".
 
- I appreciate that there is an option to not include the Godot logo in projects, but wouldn't it be better if everyone wanted to proudly display it? 

Maybe you know people that do proudly display the logo, and like it, but I have seen some comments otherwise (including multiple issues that were shut down). Then again, it's probably not helpful to argue anecdotes here. 

I will just say that this discussion exists in critique of the logo, it's not the first such discussion, and I am glad the discussion is being had.

**Comment by AThousandShips** [comment]
I just want to voice some of my serious concerns that the features of the current icon are framed as "unprofessional" as a number of them, like the softer jawline and larger eyes, are traditionally seen as "feminine" features, branding the current one as unprofessional borders on misogyny and I find it offensive

I will of course attribute it to uncritically applying norms, but it worries me that it hasn't been considered and been a part of thinking for design values in casting it as more professional

**Reply by CarpenterBlue** [reply]
This feels dishonest when the goal is not to remove or add gendered features off-of genderless robot but to substitute unaesthetic curves and un-even distribution of details and lack of cohesive composition.

**Reply by AThousandShips** [reply]
Can't be done without recognizing that the changes very obviously adds conventionally masculine traits

Eveything else is personal taste, but these are actual social norms, where conventionally masculine traits are considered more professional than conventionally feminine

But if people are going to brand genuinely raising issues over entrenched sexist norms in design and aesthetics then I'm out...

**Reply by Flavelius** [reply]
If that's your basis for arguing about potential logo redesigns then I think you should be all for a change, as the current logo is almost all blue which is conventionally targeting the more masculine attribution, and also missing the representation of all other equally important colors (and shapes as well).

**Reply by AThousandShips** [reply]
Just keep ignoring my points then, fine, good day to you, I'm not going to stay here and be mocked

All I'm asking is for people to challenge their perceptions and norms and question why they perceive certain things in certain way, but I know that can be hard and uncomfortable

**Reply by Flavelius** [reply]
I think my proposal not just _not_ ignored your points, that you make about feminine features, but can also be seen as being about them. I made the invisible compounding shape a little more round, the spikes wider an a little more balanced and the jawline just a tad softer.
A logo for a game engine, in my opinion, only needs to represent the technical thing it is about, it can do more and i'm not against that.
Edit: and to the point of professionalism, it's about the derpiness; and that is not based on its feminine features, more it's imbalance.

**Reply by dalexeev** [reply]
_Disclaimer. Potentially sensitive topic, please follow the Code of Conduct, respect people and assume positive intentions._

Personally, I haven't thought about the gender of the Godot logo until now. The blue color is often associated with masculine characters, but other than that I don't see any obvious gender cues in the logo, either feminine or masculine (such as headdress, hair or facial hair). Face shape can vary significantly in both men and women. Narrower cheekbones are not necessarily an attribute of a male character.

Perhaps this is why many projects choose abstract shapes, non-anthropomorphic animals and robots as a logo and/or mascot. Historically, Godot logo is an anthropomorphic robot. I think the character looks like gender neutral, unlike derivative non-official characters like Godette. Although in official announcements we used phrases like "Mr. Godot told me to tell you he won't come this evening but surely tomorrow." The engine name also refers to Samuel Beckett's play "Waiting for Godot".

![](https://github.com/godotengine/godot-proposals/assets/47700418/4b45b4fc-77c0-4946-89b5-8f2f43765a62)

Therefore, without advocating changing the logo to a "more professional" one, it still seems to me that the gender argument is unfounded. Changing the logo is not necessarily about adding/strengthening masculine traits. "More professional" does not mean "more masculine" traits, I don't think that's what people meant here. In my opinion, their concerns and objections are different (professional/marketing/commercial vs garage-made/childish/shameful).

**Reply by AThousandShips** [reply]
I'm going to clarify my points as much as I can, but given that me raising points about norms, stereotypes, and sexism resulted in being declared "dishonest" I genuinely do not feel comfortable discussing this topic in the present company:

The changes being labelled as "more professional" substitute traits conventionally gendered feminine, such as larger eyes, and the general shape of the face being longer and narrower, with traits often coded masculine.

My point was to raise concern over the uncritical associations of such, the fact that design decisions are often done with little critical thought of what they genuinely code one way or another, which is a serious issue across many fields. The actual supposed gender of the character isn't the issue, but the association of certain traits with certain value judgements.

This isn't about the very real variation in traits among humans, but the social norms associated with certain traits. 

And my main concern is the immediate rejection and refusal to consider these points, but instead declare them as "dishonest". And the fact that these concerns were dismissed out of hand when raised by a woman pointing out what was perceived and questioned as signs of uncritical application of sexist norms.

I'm asking the people realise that labels such as "professional" or "unprofessional" doesn't exist in a vacuum but in a world of norms, preconceptions, stereotypes, and prejudices. And that one must analyse that crucially when assigning such labels.

**Reply by Zireael07** [reply]
Larger eyes are not necessarily feminine coded - they are often, uh, child coded? Baby coded? (Not a native speaker of English)

You get what I mean. And baby or child associated things are then often associated with unprofessionalism, but that's veering into offtopic area imho as it's a secondary observation "big eyes -> baby -> unprofessional"

**Reply by CarpenterBlue** [reply]
`being declared "dishonest" I genuinely do not feel comfortable discussing this topic in the present company:`
I will see myself out then, I do not like causing people discomfort, I would advise same to you as you need to work on your conduct. As it stands we are both detriment to the civil discussion. I likewise do not feel comfortable continuing the discussion in the present company as I do not like when people imply that I am being misogynist for merely discussing logos. Such harsh words shouldn't be used without proper reason. Moreover you seem to have something against Flavius in general but that may be just my perception as I am already bit on the edge from your behaviour myself as I noticed you un-liked my cat-godot edit when I said something which you didn't like.

I used word "dishonest" because it felt like you tried shutdown discussion of people (namely Me and Flavelius) which you personally don't like by bring in gender into it. It feels like abuse of code of conduct.

Nonetheless, I extend my apologies if I made you feel.

I will try to remedy the situation by properly addressing your concerns, I gave the pictures look and the eye-size stays the same, though I could see how you came to that conclusion as they jump a little bit in the gif and the head is made narrower to have even spacing with the nose and the edge and the pupils changed the focus so technically in the end they are proportionally larger. The chin was also made less broad which is actually removal of masculine trait.

And finally to properly define my understanding of "professional" it has nothing to do with with gender. What I am after is to have cleaner designed logo using even proportions. namely, the cog part of the head and facial proportions.

This is my last post here for very long while as I do not feel welcome and the discussion here is very emotionally charged.

**Reply by AThousandShips** [reply]
> you un-liked my cat-godot edit when I said something which you didn't like.

I deal with dozen to hundreds of people on here ever single day, I don't have time to form personal vandettas with anyone, please get over yourself, I didn't even mention you until you attacked me

**Reply by Zireael07** [reply]
@CarpenterBlue That reads like an unwarranted personal attack. @AThousandShips mostly "deals" with people by asking basic questions such as what their hardware specs are and what they are trying to accomplish and please provide a MRP, as many bug reports turn out to be user errors.

**Reply by xdega** [reply]
@AThousandShips This response is very confusing. Nobody was bringing gender into any of this, and if they did I am certain it would be rightfully rejected.

The "professionalism" of the art style is nothing to do with gender, but to do with the poor quality of of the drawing itself at a fundamental level. 

It looks like someone who has no idea how to draw a clean Bézier curve (or use the pen tool in Illustrator), simply grabbed basic shape tools and started drawing random shapes until they had something that resembled a "character". 

You only have to look at @Dark-Horizons below, to see how much more "cohesive" a logo can be when there is some thought behind the composition, contrast, etc (all design considerations).

**Comment by Dark-Horizons** [comment]
Since the Godot logo includes a character, it differs from its counterparts such as Unity or Unreal Engine, and I think this is very valuable. 
That's why I tried to stick to the old design tightly.

This study is only a suggestion.

![image](https://github.com/godotengine/godot-proposals/assets/110924254/579c8b32-5fca-4b5c-b66b-e9d186641f06)
For Details: https://www.artstation.com/artwork/xDoeA2

**Reply by xdega** [reply]
This is precisely what I a talking about when I talk about a good "modern" logo design. It has good composition, color, contrast, etc, and really highlights where the current logo falls short in terms of production quality.

**Reply by betalars** [reply]
Okay wow, I didn't think I would ever say this, but this is unironically a Logo Redesign I could get behind. I think I'm giving this Idea my own shot, but dang does this look cool and still true to the original.

**Comment by betalars** [comment]
Still thinks the godot logo does not look serious enough. Made a few small adjustments myself.
![blogcat godot mashup spelled blobot](https://github.com/user-attachments/assets/33bd1eb6-9cbd-4368-801a-c61c1870b8b3)
